float64 m1_sim
float64 m2_sim
float64 thres_1
float64 thres_2
float64 thres_obj
int64 model_n
int64 cluster_n
bool obj_detect

